__tests__/app.test.js

```diff
@@ -166,5 +212,117 @@ describe('/api/articles', ()=>{
             })
         })
     })
-    
-})
\ No newline at end of file
+    describe('POST /api/articles/:article_id/comments valid endpoints', ()=>{
+        test('201 Adds a new comment object to the database', ()=>{
+            const newComment = {
+                "username": "butter_bridge",
+                "body": "I can't wait for bed-time"
+            }
+            return request(app)
+            .post("/api/articles/5/comments")
+            .send(newComment)
+            .expect(201)
+            .then(() => {
```

Comment:
Ideally we should be asserting against the response that comes back after posting. If we have a RETURNING * in our post we will get a new comment back which we can assert against 👍